build/react-gallery-swiper.js

```diff
@@ -187,7 +187,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	            var newImages = nextProps.images,
 	                newLazyload = nextProps.lazyLoad;
 
-	            if (images !== newImages) {
+	            if (images !== newImages || images.length !== newImages.length) {
```

Comment:
is there a use case for this? Are you changing the original list somehow instead of providing a new list?